packages/ui/src/Card.module.scss

```diff
@@ -0,0 +1,47 @@
+@use '../styles/constants' as c;
+
+.wrapper {
+  display: flex;
+  flex-direction: column;
+  align-content: stretch;
+  box-shadow: 0 c.$cardShadowSize c.$cardShadowBlur 0 c.$colorShadow;
```

Comment:
Should this be called `c.$cardShaddowColor` instead of `c.colorShadow`?

Reply:
I just followed the convention for color names. We don't have any component-specific colors yet and I don't think we'll have multiple shadow colors. I'd rather keep this name, WDYT?